lib/Optimizer/GraphOptimizer/GraphOptimizer.cpp

```diff
@@ -62,6 +62,15 @@ static bool shouldDeleteNode(Node *N) {
   return true;
 }
 
+static bool shouldDeleteConstants(Function *F) {
+  Module *pMod = F->getParent();
+  bool state = true;
+  for (auto F : pMod->getFunctions()) {
+    state = state & (F->getState() >= FunctionState::FuncLoaded);
```

Comment:
nit: If we keep this, we could just do `if (F->getState() < FunctionState::FuncLoaded) return false` and then return true at the end of the func.